packages/flutter_tools/lib/src/isolated/resident_web_runner.dart

```diff
@@ -81,7 +81,7 @@ const String kExitMessage = 'Failed to establish connection with the application
 
 class ResidentWebRunner extends ResidentRunner {
```

Comment:
You can get of a lot of `!` in this file if `_fileSystem` and `_logger` were correctly marked not nullable.
```dart
  final FileSystem _fileSystem;
  final Logger _logger;
```

Reply:
and in `resident_runner`
https://github.com/flutter/flutter/blob/5805f4544534ccb3200e50625f1dd79cd9881b1f/packages/flutter_tools/lib/src/resident_runner.dart#L1087-L1090

Reply:
done